packages/monaco/src/browser/monaco-quick-open-service.ts

```diff
@@ -26,6 +26,7 @@ export class MonacoQuickOpenService extends QuickOpenService {
     protected _widget: monaco.quickOpen.QuickOpenWidget | undefined;
     protected opts: MonacoQuickOpenControllerOpts | undefined;
     protected previousActiveElement: Element | undefined;
+    private lookingFor: string = "";
```

Comment:
I would suggest renaming this to `lastLookedFor`, to show that it keeps the value of the last type.

Reply:
Done